Noura_Alsubaie/week_06/day_02/seller_gate/config/routes.rb

```diff
@@ -0,0 +1,8 @@
+Rails.application.routes.draw do
+  
+  get 'categories/index'
+  get 'categories/show'
```

Comment:
So you don't need these `get` routes if you have `sources` in place, it handles all of that for you!